brayns/engine/Scene.cpp

```diff
@@ -151,7 +151,12 @@ size_t Scene::addModel(ModelDescriptorPtr modelDescriptor)
 
     model.setBVHFlags(defaultBVHFlags);
     model.buildBoundingBox();
-    model.commitGeometry();
+
+    // Since models can be added concurrently we check if that is supported
+    if (supportsConcurrentSceneUpdates())
+        model.commitGeometry();
+    else
+        markModified();
```

Comment:
Shouldn't markModified be called anyway? Looking at the code in Scene::commit() I can't see how the scene bounding box gets updated otherwise.